wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -223,7 +223,7 @@ int atmel_ecc_alloc(int slotType)
                 slotId = ATECC_SLOT_ENC_PARENT;
                 break;
             case ATMEL_SLOT_ANY:
-                for (i=0; i < ATECC_MAX_SLOT; i++) {
+                for (i=0; i <= ATECC_MAX_SLOT; i++) {
```

Comment:
This is a zero based index and max slot should be `<` only. Please revert here and at 263.

Reply:
There is one more place, in atmel.h where ATECC_MAX_SLOT is defined, it says:
       #define ATECC_MAX_SLOT      (0x7) /* Only use 0-7 */
I think the value needs to also be changed to 8 or the comment changed to 0-6.